docs/xks/developer-guide/report.md

```diff
@@ -0,0 +1,57 @@
+---
+id: reports
+title: Reports
+---
+
+As a XKS 2.0 customer you are getting monthly reports, these reports exist to help you as a customer
+to get a quick overview of your environment and how your platform is doing.
+
+## Security
+
+Xenit **isn't** responsible for the security of your application but a part of the report we point to
+the number of CVE:s in your applications found by our automated scanning tools that we have in XKF.
+
+This is a extra way for at Xenit to help you as a developer but you **can't** rely on Xenit to point out specific security flaws in your application. Security is something that **you and your company** is in charge of we are just trying to visualize some of the data that we have access to.
+
+As a part of XKF we scan all images continuously that is running inside our clusters, todo this we depend on
+[starboard](https://github.com/aquasecurity/starboard/) together with [trivy](https://github.com/aquasecurity/trivy/).
+
+If you are using Xenits CI solution you are by default scanning your container images with trivy at creation time
```

Comment:
Suggestion:

If you are using the Xenit CI solution you are by default scanning your container images with Trivy at creation time
but new CVE:s might have come out since.... 

continues --> continuous